### DISCUSSION TITLE
Cannot connect

### DISCUSSION THREAD

**Discussion by qwobii** [iterate-ch/cyberduck#14764] [Q&A]
I cannot connect to my webspace and none of my bookmarks are shown when loading the app. 
![Capture](https://github.com/iterate-ch/cyberduck/assets/135703816/68d59488-4716-438a-80e9-dbe792b1eb8d)

**Comment by tatze96** [comment]
Regarding the connection error:
Please try to resolve the issue by increasing the "Timeout for opening connections" within the Cyberduck preferences *Connection* tab.

If this doesn't resolve the error:
Can you please [enable debug logging](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level), try to connect to the server, and send us the log file afterward? If you don't like to see sensitive data to be shared publicly, please send the log file with the ticket reference GH-14764 to support@cyberduck.io, otherwise attach it here.

**Sidenote**: If you want to share the log file here, make sure to attach it through the GitHub web interface, as it wouldn't be visible to us otherwise.

Regarding the non-visible Bookmarks:
Do you use the standalone version of Cyberduck from our website or the version out of the Windows Store?

Please verify that there are bookmark files available in the *Bookmarks* folder within the [application support folder](https://docs.cyberduck.io/cyberduck/support/#application-support-folder).
